<launch>
  <arg name="linear_throttle_factor" value="1.0" />
  <arg name="angular_throttle_factor" value="1.0" />
  <arg name="finger_throttle_factor" value="1.5" />
  
  <node name="jaco_joy_teleop" pkg="jaco_teleop" type="jaco_joy_teleop" output="screen">
    <param name="linear_throttle_factor" type="double" value="$(arg linear_throttle_factor)" />
    <param name="angular_throttle_factor" type="double" value="$(arg angular_throttle_factor)" />
    <param name="finger_throttle_factor" type="double" value="$(arg finger_throttle_factor)" />
    <remap from="joy" to="arm/joy" />
  </node>
</launch>
